app/src/main/res/layout/feeditem_fragment.xml

```diff
@@ -171,6 +171,20 @@
             android:layout_height="1dp"
             android:background="?android:attr/dividerVertical"/>
 
+        <TextView
+            android:id="@+id/noMediaMsg"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="visible"
+            android:textAlignment="center"
+            android:background="@color/light_gray"
+            android:text="@string/noMediaMsgText"/>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="?android:attr/dividerVertical"/>
```

Comment:
This means that feed items with media now have two separators. Could you please try to remove one of them and post a screenshot? Maybe it already looks okay with only one.

Reply:
missed that, here are screenshots with one divider below noMediaLabel

![Screenshot_20201124_152704](https://user-images.githubusercontent.com/17114678/100163326-4725c200-2e6a-11eb-91cb-32bdc7ba5f22.png) ![Screenshot_20201124_152837](https://user-images.githubusercontent.com/17114678/100163327-48ef8580-2e6a-11eb-9eb1-8ee05458c6a6.png)

Reply:
Could you please change the background color to `android:attr/dividerVertical`? That should make the contrast better. Currently, it is pretty hard to read.

Reply:
Changed the background.  I'm not sure if it's typical to always post screenshots when a visual change is made.  Seems like it might be undesirable to fill up threads with images for little changes like this? Then again, if something like readability is in question, I should?